test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -17,30 +17,92 @@
 package controllers
 
 import base.SpecBase
-import play.api.test.FakeRequest
+import controllers.actions.FakeAuthoriseAction
+import controllers.helpers.CheckYourAnswersHelper
+import models.TraderGoodsProfile
+import org.mockito.ArgumentMatchers.any
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
 import play.api.test.Helpers._
+import uk.gov.hmrc.govukfrontend.views.Aliases.{ActionItem, Actions, HtmlContent, SummaryListRow, Value}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Key
 import viewmodels.govuk.SummaryListFluency
 import views.html.CheckYourAnswersView
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar.mock
 
 class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency {
+  val checkYourAnswersView: CheckYourAnswersView         = app.injector.instanceOf[CheckYourAnswersView]
+  val mockCheckYourAnswersHelper: CheckYourAnswersHelper = mock[CheckYourAnswersHelper]
 
-  "Check Your Answers Controller" - {
+  val checkYourAnswersController = new CheckYourAnswersController(
+    stubMessagesControllerComponents(),
+    new FakeAuthoriseAction(defaultBodyParser),
+    checkYourAnswersView,
+    sessionRequest,
+    mockCheckYourAnswersHelper
+  )
+
+  "CheckYourAnswersController" - {
 
     "must return OK and the correct view for a GET" in {
 
-      val application = applicationBuilder(userAnswers = emptyUserAnswers).build()
+      val summaryList = List(
```

Comment:
We could shorten this using the helper and creating answers... but this is fine as is and it could be argued that this keeps helper and controller test seperate.

Reply:
i like to separate things - we could have separate test-utils file called test-gubbins or the like where we can store the huge variables

will keep it as it is unless you really want it to change

Reply:
Sounds good! No need to change for now. If it works and is well tested, it works. - I tend to leave personal preference out of the "need to have" realm.